src/huggingface_hub/inference/_providers/hf_inference/_common.py

```diff
@@ -0,0 +1,25 @@
+from huggingface_hub.inference._common import _fetch_recommended_models
```

Comment:
At some point `_fetch_recommended_models` should be moved inside `./_providers/hf_inference/_common.py` as well